velox/functions/prestosql/ArrayFunctions.h

```diff
@@ -280,4 +280,43 @@ struct CombinationsFunction {
   }
 };
 
+template <typename T>
+struct udf_array_sum {
+  VELOX_DEFINE_FUNCTION_TYPES(T)
+  template <typename OT, typename IT>
+  FOLLY_ALWAYS_INLINE bool call(OT& out, const IT& array) {
+    if (array.mayHaveNulls()) {
+      bool allNulls = true;
+      OT sum = 0;
+      for (const auto& item : array) {
+        if (item.has_value()) {
+          allNulls = false;
+          sum += *item;
+        }
+      }
+      out = sum;
+      return true;
+    }
+
+    // Not nulls path
```

Comment:
Typo: "Not" -> "No" + period at the end.

Reply:
Per @laithsakka comments I am removing the no nulls path from this function as it never gets called because the callNullFree path would get called if there were no nulls.